useful-scripts/misc-test-utils/src/modules/subject/upload/index.ts

```diff
@@ -0,0 +1,226 @@
+  const metaGlob = await globby(`${cwd}/test-files/*.meta.csv`);
+  const metaGlob2 = metaGlob[0];
+
+  fs.rename(
+    `${metaGlob2}`,
+    `${cwd}/test-files/testfile-${randomId}.meta.csv`,
+    e => (e ? console.error(chalk.red(e)) : ''),
+  );
+};
+
+const addSubject = async (releaseId: string) => {
+  const glob = await globby(`${cwd}/zip-files/*.zip`);
+
+  // potential issue here with how windows paths are formatted in git bash
+  if (glob !== undefined) {
+    const oldPath =
+      process.cwd() +
+      '/zip-files/' +
+      fs.readdirSync(process.cwd() + '/zip-files')[0];
+
+    const newPath =
+      process.cwd() + '/' + fs.readdirSync(process.cwd() + '/zip-files')[0];
+    fs.renameSync(oldPath, newPath);
+  }
+
+  const rootGlob = await globby(`${cwd}/clean-test-zip-*.zip`);
+  const form = new FormData();
+  form.append('zipFile', fs.createReadStream(rootGlob[0]));
+
+  try {
+    const res = await axios({
+      maxContentLength: Infinity,
+      maxBodyLength: Infinity,
+      method: 'POST',
+      url: `${ADMIN_URL}/api/release/${releaseId}/zip-data?name=importer-subject-${v4()}`,
+      data: form,
+      headers: {
+        ...form.getHeaders(),
+        Authorization: `Bearer ${JWT_TOKEN}`,
+      },
+    });
+    const subjectId = res.data.id;
+    return subjectId;
+  } catch (e) {
+    errorHandler(e);
+  }
+};
+
+const getSubjectProgress = async (releaseId: string, subjectId: string) => {
+  try {
+    const res = await axios({
+      method: 'GET',
+      url: `${ADMIN_URL}/api/release/${releaseId}/data/${subjectId}/import/status`,
+      headers: {
+        Authorization: `Bearer ${JWT_TOKEN}`,
+        'Content-Type': 'application/json',
+      },
+    });
+    return res.data.status;
+  } catch (e) {
+    errorHandler(e);
+  }
+};
+
+const getSubjectIdArr = async (releaseId: string) => {
+  try {
+    const res = await axios({
+      method: 'GET',
+      url: `${ADMIN_URL}/api/release/${releaseId}/meta-guidance`,
+      headers: {
+        Authorization: `Bearer ${JWT_TOKEN}`,
+        'Content-Type': 'application/json',
+      },
+    });
+    const subjects: SubjectArray[] = res.data?.subjects;
+    let subjArr: { id: string; content: string }[] = [];
+    subjects.forEach(sub => {
+      subjArr.push({ id: `${sub.id}`, content: `Hello ${v4()}` });
+    });
+    return subjArr;
+  } catch (e) {
+    errorHandler(e);
+  }
+  return;
+};
+
+const addMetaGuidance = async (
+  subjArr: { id: string; content: string }[],
+  releaseId: string,
+) => {
+  try {
+    await axios({
+      method: 'PATCH',
+      url: `${ADMIN_URL}/api/release/${releaseId}/meta-guidance`,
+      headers: {
+        Authorization: `Bearer ${JWT_TOKEN}`,
+        'Content-Type': 'application/json',
+      },
+      data: {
+        content: '<p>testing</p>',
+        subjects: subjArr,
+      },
+    });
+  } catch (e) {
+    errorHandler(e);
+  }
+};
+
+const UploadSingleSubject = async (releaseId: string) => {
```

Comment:
Functions should be camelCased i.e. `uploadSingleSubject`. The main exception are React components, which are PascalCased instead.